src/components/data-management/DownloadData.jsx

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable import/no-unresolved */
+import React, { useEffect } from 'react';
+import _ from 'lodash';
+import {
+  Menu, Tooltip, Dropdown, Button,
+} from 'antd';
+
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { saveAs } from 'file-saver';
+import downloadTypes from 'utils/data-management/downloadTypes';
+import { getFromApiExpectOK } from 'utils/getDataExpectOK';
+import pushNotificationMessage from 'utils/pushNotificationMessage';
+import endUserMessages from 'utils/endUserMessages';
+import pipelineStatus from '../../utils/pipelineStatusValues';
+import { exportQCParameters, filterQCParameters } from '../../utils/data-management/exportQCParameters';
+import { loadBackendStatus } from '../../redux/actions/backendStatus/index';
+
+const DownloadData = (props) => {
+  const {
+    activeProjectUuid,
+  } = props;
+  const dispatch = useDispatch();
+  const activeProject = useSelector((state) => state.projects[activeProjectUuid]);
+  const experimentSettings = useSelector((state) => state.experimentSettings);
+  const backendStatus = useSelector((state) => state.backendStatus);
+  const samples = useSelector((state) => state.samples);
+  const projects = useSelector((state) => state.projects);
+
+  // Change if we have more than one experiment per project
+  const experimentId = activeProject?.experiments[0];
+
+  useEffect(() => {
+    if (experimentId && !backendStatus[experimentId]) {
+      dispatch(loadBackendStatus(experimentId));
+    }
+  }, [experimentId]);
+
+  const getQcHasRun = () => (
+    experimentId
+    && (backendStatus[experimentId]?.status.pipeline?.status === pipelineStatus.SUCCEEDED)
+  );
+
+  const getGem2sHasRun = () => (
+    experimentId
+    && (backendStatus[experimentId]?.status?.gem2s?.status === pipelineStatus.SUCCEEDED)
+  );
```

Comment:
These should be in a `useState` and `useEffect` should update the state on changes to `backendStatus` instead of the current operation -- this causes unnecessary rerenders as is.